app/models/area.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+# Monkeypatch clarat_base Area
+require ClaratBase::Engine.root.join('app', 'models', 'area')
+# Bounding Box around an area that a non-personal offer provides service to.
+class Area < ActiveRecord::Base
+  # Associations
+  has_many :offers, inverse_of: :area
```

Comment:
Habe alles verändert bis auf City - da war schon vorher eine Association in claradmin, die in clarat_base nicht ist.